packages/tracing/src/integrations/node/ioredis.ts

```diff
@@ -0,0 +1,101 @@
+import { Hub } from '@sentry/hub';
```

Comment:
You'll have to update `packages/tracing/src/hubextensions.ts` and `packages/tracing/src/integrations/index.ts` as well